src/item.rs

```diff
@@ -838,6 +851,14 @@ impl Item {
             #[cfg(feature = "ck3")]
             Item::Suggestion => "common/suggestions/",
             #[cfg(feature = "ck3")]
+            Item::TaxSlot => "common/tax_slots/",
```

Comment:
Hmm that's a parent directory with only subdirectories in it. What's the difference between TaxSlot and TaxSlotType and where are TaxSlot items really defined?